fmidev/terraform-aws-geoweb
# Fix zalando config & adjust default backup values to make more sense

* Changed default backup schedule and retention times
  * Retention time is 2 weeks or 14 days for both backups
  * Basebackup is taken once everyday at midnight (or 2:00 due to timezone to be exact)
  * Logical backup is taken 4 times a day
* Removed WALG_S3_PREFIX and CLONE_WALG_S3_PREFIX values, as those were forcing the basebackups of all databases to same location, causing problems.
* Removed kube-system namespace from pod-config, as it's in the same namespace that can be modified by user, this should work in all cases

How to test:
* Code review
* Applications are running in a test EKS cluster (named test-geoweb in intlgeoweb AWS accout) created with these changes and Zalando databases (with these helm-chart changes https://github.com/fmidev/helm-charts/pull/58) in https://terraform.opengeoweb.com.  Assume role and access the cluster to inspect the databases.
* Backups can be reviewed in AWS s3 bucket named zalando-test-db-backups, note that the logical backup schedule was only just now modified from once every 4th hour to once every 6th hour so the old backups are done with that schedule.
